lib/openshift/local_volume.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/project_resource'
+
+module BushSlicer
+  # represents LocalVolume CRD
+  class LocalVolume < ProjectResource
+    RESOURCE = 'localvolume'
```

Comment:
please use full resource path and API name. most likely `localvolumes.openshift.something...`

Reply:
look at `selfLink` for exact string

Reply:
Thanks for the review, updated.

Reply:
did you forget to push the change? I still see the old string here.